src/main/java/org/folio/config/ConfigurationsClient.java

```diff
@@ -155,6 +153,8 @@
         headers.put(OkapiHeaders.OKAPI_TOKEN_HEADER, okapiHeaders.getToken());
 
         try {
+          verifyOkapiHeaders(okapiHeaders);
```

Comment:
Seems this line is not needed here because earlier checkEntry() already does the check.